simphony/testing/abc_check_lattice.py

```diff
@@ -22,7 +22,7 @@ def setUp(self):
         self.addTypeEqualityFunc(
             LatticeNode, partial(compare_lattice_nodes, testcase=self))
         self.size = (5, 10, 15)
-        self.base_vect = (0.1, 0.2, 0.3)
+        self.base_vect = (0.2, 0.2, 0.2)
         self.origin = (-2.0, 0.0, 1.0)
         self.container = self.container_factory(
             'foo', 'Cubic', self.base_vect, self.size, self.origin)
```

Comment:
It does not have any practical implication, but as "foo" token has been changed for "my_name" in the rest of the file this should change as well